oem/firstvoices/common/build_keyboards.sh

```diff
@@ -58,10 +81,12 @@ while [[ $# -gt 0 ]] ; do
 done
 
 # Clean existing Keyboards folder
+# Can't remove entire folder because Android has other assets/
 
 if [ $DO_CLEAN = true ]; then
   echo "Cleaning target path $KEYBOARDS_TARGET"
-  rm -rf $KEYBOARDS_TARGET/*
+  rm -rf "$KEYBOARDS_TARGET/*.kmp"
+  rm -rf "$KEYBOARDS_TARGET/*.keyboard_info"
```

Comment:
```suggestion
  rm -f "$KEYBOARDS_TARGET/*.kmp"
  rm -f "$KEYBOARDS_TARGET/*.keyboard_info"
```

Reply:
Also can we add a test add top of file that `$KEYBOARDS_TARGET` is not empty string so that this doesn't attempt bad things in wrong places (even if it would be a no-op).

Reply:
I notice this won't clean if the previous build copied/built keyboards 
```
$KEYBOARDS_TARGET/*/*.js
$KEYBOARDS_TARGET/*/kmp.json
```

Reply:
Yeah, don't worry about this because it's going to come out in the wash with the iOS kmp conversion.

Reply:
It's a side effect that $KEYBOARDS_TARGET should be different when building vs downloading keyboard
### building
/app/src/main/assets/packages/*sil_euro_latin/sil_euro_latin.js*
/app/src/main/assets/packages/*sil_euro_latin/kmp.json*

### downloading
/app/src/main/assets/packages/*fv_all.kmp*